src/videodownlinkbandwidthpolicy/AllHighestVideoBandwidthPolicy.ts

```diff
@@ -15,6 +16,9 @@ import VideoDownlinkBandwidthPolicy from './VideoDownlinkBandwidthPolicy';
 export default class AllHighestVideoBandwidthPolicy implements VideoDownlinkBandwidthPolicy {
   private optimalReceiveSet: VideoStreamIdSet;
   private subscribedReceiveSet: VideoStreamIdSet;
+  private areVideoSourcesChosen: boolean;
+  private videoSources: VideoSource[];
```

Comment:
Can we just do videoSources to be VideoSources[]| undefined? Below we can just check whether videoSources !== undefined and not need `areVideoSourcesChosen` variable.

Reply:
will do